src/services/TechRecordsService.ts

```diff
@@ -107,22 +137,138 @@ class TechRecordsService {
       });
   }
 
-  public updateTechRecord(techRecord: ITechRecordWrapper) {
-    return this.techRecordsDAO.updateSingle(techRecord)
-      .then((data: any) => {
-        const response = data.Attributes;
-        const vrms = [{vrm: response.primaryVrm, isPrimary: true}];
-        Object.assign(response, {
-          vrms
+  public updateTechRecord(techRecord: ITechRecordWrapper, msUserDetails: any, files?: string[]) {
+    if (files && files.length) {
+      const promises = [];
+      for (const file of files) {
+        promises.push(this.uploadFile(file, techRecord.vin));
+      }
+      return Promise.all(promises)
+        .then((uploadedData: ManagedUpload.SendData[]) => {
+          const documents: string[] = [];
+          if (uploadedData && uploadedData.length) {
```

Comment:
sonar says no test for negative case here. 
![image](https://user-images.githubusercontent.com/50110006/70146753-c12dcb80-169a-11ea-81b0-9c2149f01328.png)

Reply:
I will ignore this as discussed with Kevin